brokerapi/brokers/bigtable/definition.go

```diff
@@ -33,7 +32,7 @@ func ServiceDefinition() *broker.ServiceDefinition {
 	}
 
 	return &broker.ServiceDefinition{
-		Name: models.BigtableName,
+		Name: "google-bigtable",
```

Comment:
In other definitions where you declare the name in a `const` it seems the only other place it's referenced is in `brokers_test.go`. Is there a reason these service definitions w/ inlined values don't appear in that test?

Reply:
That test file is still one of the originals, I think the original author chose to pick a few services that they felt were representative of the cases.

I'm hoping to move it over to just being mocked out today so we can get rid of the need to reference anything by name. That change should also shore up the tests for our main execution path so we can upgrade that vulnerable library that led to #361.